tensorflow_quantum/datasets/spin_system_test.py

```diff
@@ -93,23 +101,120 @@ def test_returned_objects(self):
                 self.assertIsInstance(addinfo[n], SpinSystemInfo)
 
     def test_param_resolver(self):
+        """Test that the resolved circuits are correct."""
+        for nspins in self.supported_nspins_tfi_chain:
+            circuits, _, _, addinfo = self.data_dict_tfi_chain[nspins]
+            for n in self.random_subset_tfi_chain:
+                resolved_circuit = cirq.resolve_parameters(
+                    addinfo[n].var_circuit, addinfo[n].params)
+                state_circuit = cirq.Simulator().simulate(
+                    circuits[n]).final_state
+                state_resolved_circuit = cirq.Simulator().simulate(
+                    resolved_circuit).final_state
+                self.assertAllClose(np.abs(
+                    np.vdot(state_circuit, state_resolved_circuit)),
+                                    1.0,
+                                    rtol=1e-3)
+
+
+class XXZChainTest(tf.test.TestCase):
+    """Testing xxz_chain."""
+    # pylint: disable=C0103
+
+    @classmethod
+    def setUpClass(self):
```

Comment:
Can you be sure to include a call to the superclasses `setUpClass` function as well ?